guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+
+### Transactional Email
+{:.no_toc}
+
+* The client’s brand colors should be ubiquitous and consistently presented in transactional emails
+* The from address of transactional emails should be the intended email address
+* Transactional emails received should have the intended imagery and body copy
+* Social links and features should be associated with the brand’s social channels and handles, i.e., should mention brand social handle, correct social URL, etc.
+* Transactional emails should contain linked products
+* Product links should navigate to the indented product when tapped/clicked
+
+### Campaign Email
+{:.no_toc}
+
+* The client’s brand colors should be ubiquitous and consistently presented in campaign emails
+* The from address of campaign emails should be the intended email address
+* Campaign emails received should have the intended imagery and body copy
+* Social links and features should be associated with the brand’s social channels and handles, i.e., should mention brand social handle, correct social URL, etc.
+
+---
+
+## Simulated Shopping Environment
+
+A portion of the QA process should mimic a shopping and browsing session for a typical shopper. There should be no console errors, no 404s and no feature failures. The site should behave as expected to ensure a completed shopping session.
+
+The user should be able to accomplish these tasks without refreshing, receiving an error, or having to stop to ask questions. Each of these events should be unblemished by long load times or missing/late loading elements. All current data points towards aiming for a sub 3 second load time for all pages, mobile and desktop. Optimal load times for conversion range between 1 and 2.7 seconds.
+
+> These impacts were felt even on tablets and mobile, with mobile being the most affected by delays. For mobile users, the optimal load time – which correlated to a 14.1% bounce rate – was 700 milliseconds. At 1.7 seconds, bounce rate was 21% – representing an increase of almost 50%. At 2.7 seconds, the bounce rate was almost 29% – representing a 103% increase.
+>
+> <cite><a target="_blank" href="https://www.akamai.com/us/en/multimedia/documents/report/akamai-state-of-online-retail-performance-spring-2017.pdf">The State of Online Retail Performance, Spring 2017, Akamai</a></cite>
+
+In addition to these common flows working, we should verify the presence of the Klaviyo or Engine actions occurring and appearing in the Klaviyo or Engine timeline.
+
+### Simulations to Run
+{:.no_toc}
+
+#### Email Sign up
+{:.no_toc}
+
+In a new incognito window (or private browsing session, etc.), a user should verify that they are not cookied by an email signup action. On a site page with an email signup form (i.e. a popup, an on-page form, etc.), the user should submit their email address and should verify that their browser has been successfully cookied.
```

Comment:
I believe that "i.e." should be changed to "e.g." here since we are giving actual examples of the signup forms, but do correct me if I'm wrong.